src/deploy/NVA_build/Tests.Dockerfile

```diff
@@ -13,11 +13,20 @@ ENV TEST_CONTAINER true
 ##############################################################
 
 # python-virtualenv python-devel libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel -- these are required by ceph tests
-RUN yum install -y -q ntpdate vim centos-release-scl && \
-    yum install -y -q rh-mongodb36 && \
-    yum install -y -q python-virtualenv python-devel libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel && \
-    yum install -y -q git && \
-    yum clean all
+# RUN dnf install -y ntpdate vim && \
+COPY ./src/deploy/NVA_build/set_mongo_repo.sh /tmp/
+RUN chmod +x /tmp/set_mongo_repo.sh && \
+    /bin/bash -xc "/tmp/set_mongo_repo.sh"
+
+RUN dnf install -y -q vim \
```

Comment:
Why do we need vim?!

Reply:
To edit scripts inside a running pod. Allows us to make patches inline very quickly to test assumptions or fix complex situations.

Reply:
I can't even start to explain how valuable this is.

Reply:
The vim is only on the test container, it will not affect the core image. 
on the test container, we extend the noobaa-core image and adding some tools for easier debug + tools for mocking our flows (such as adding mongo to the image) so we will be able to run all the unit tests as a single docker run (without relying on and kubernetes env)